### DISCUSSION TITLE
What if datatlas was a monorepo ?

### DISCUSSION THREAD

**Discussion by lutangar** [datatlas-erasme/front#140] [Ideas]
Since everything is code in Javascript/Typescript, __datatlas__ could be reorg as a __monorepo__ :
    - https://github.com/datatlas-erasme/tools
    - https://github.com/datatlas-erasme/back
    - https://github.com/datatlas-erasme/front

I could list pros/cons myself, some people at [Nrwl](https://nrwl.io/), the company behind [Nx](https://nx.dev/) already did a great job there :
https://monorepo.tools/

*This is just a discussion, there's no rush doing anything but it would be great if you could read this...*

**Comment by aangelot** [comment]
Hey, thanks for the sharing, Nrwl did a great job of explaining and comparing.

I'll be happy to discuss this with the dev team at our next instance.
I can see the benefits, but I can't predict the cost of switching to monorepo, your feedback will be useful.

**Comment by OlivPi** [comment]
It's could be a nice solution for the project.

I also saw this solution : https://turborepo.org/
